Server/VikingVault.Services/TimedPaymentService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using VikingVault.DataAccess;
+using VikingVault.DataAccess.Models;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class TimedPaymentService : IHostedService
+    {
+        private Timer _timer;
+        public IServiceProvider Services { get; }
+        public int PaymentCheckInterval = 5 * 60;
```

Comment:
Is this 300 what? 300 Spartans, sheep, seconds, minutes? Please add the unit of measurement as part of the variable name to make it descriptive